styles.css

```diff
@@ -33,3 +33,45 @@ pre{
 body > div.row {
     margin-right: 0px;
 }
+
+.flex-content-container {
```

Comment:
I'm concerned about these global changes ... have you tested the rest of the pages with this?

Reply:
CSS here is mainly so it can trickle down through multiple directories for my section and for potential future use in other pages. The classes are currently not used in other pages.

Reply:
can't it be included only in your chapter directory then (e.g. overriding the main css)?